styx-scheduler-service/src/main/java/com/spotify/styx/Scheduler.java

```diff
@@ -143,6 +147,8 @@ void tick() {
                 ConcurrentHashMap::new,
                 counting()));
 
+    currentResourceUsage.forEach(stats::resourceUsage);
```

Comment:
`stats:updateResourceUsage` and so on would be better. A separated PR to keep this one smaller.